src/Components/test/E2ETest/ServerRenderingTests/FormHandlingTests/FormWithParentBindingContextTest.cs

```diff
@@ -393,7 +458,9 @@ public DispatchToForm(FormWithParentBindingContextTest test) : this()
         public string InputFieldId { get; internal set; } = "firstName";
         public string InputFieldCssSelector { get; internal set; } = null;
         public bool ShouldCauseInternalServerError { get; internal set; }
+        public bool ShouldCauseBindingErrors { get; internal set; }
```

Comment:
Minor detail, but I think we could remove `ShouldCauseBindingErrors` and just trigger the checks based on whether `AssertErrors` is nonnull. Otherwise someone might write a test that sets `ShouldCauseBindingErrors=true` without `AssertErrors`, and think it's proving there are errors, but it wouldn't actually assert anything. Or they might add `AssertErrors` without `ShouldCauseBindingErrors` and again it wouldn't assert anything.